### ISSUE TITLE
Example needs more explanations 

### ISSUE THREAD

**Issue by Tzelon** [justadudewhohacks/opencv4nodejs#166]
Hey, 
I'm new to OpenCV and the examples are very helpful but there are things that I can't feature out.
I think adding an explanation of what an example do and a small comment on each line/function will be match help to me and others that just beginning with OpenCV.

Thanks
Tzelon

**Comment by justadudewhohacks** [comment]
Fair enough. I will try to add more comments for the examples. For now I also have articles published on medium for some of the more complex examples. Maybe those might help you as well.

**Comment by Tzelon** [comment]
Yes, I saw them :) and started to read.

**Comment by justadudewhohacks** [comment]
Sure, currently have to catch up with the auto build feature, to make sure everything is settled. Will probably have time this weekend to document the examples. If you need some help urgently you can DM me on twitter with questions as well.

**Comment by Tzelon** [comment]
Nothing urgent.
Thank you for the help

**Comment by wbern** [comment]
@Tzelon Can you look at the examples, is there anything else that would be useful to add at this point?

**Comment by wilwad** [comment]
Sorry if this has been answered before: 
Is there an example on how to save/load the trained data to XML or JSON so there is no need to train the recognizers? Thank you in advance.
